PowerShell.Common.props

```
@@ -110,6 +110,21 @@
     <HighEntropyVA>true</HighEntropyVA>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <StronglyTypedOutputFolder>gen</StronglyTypedOutputFolder>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- We skip this for Microsoft.Management.UI.Internal because we have to use a workaround for WPF project. -->
+    <EmbeddedResource Update = "resources/*.resx" Condition = "'$(MSBuildProjectName)' != 'Microsoft.Management.UI.Internal'">
```

Is this format correct? I'm not sure how tolerant the format is with spacing around `=`; you can see on line #130 the `Condition` entry has no space between it and the `=` sign.

```suggestion
    <EmbeddedResource Update="resources/*.resx" Condition="'$(MSBuildProjectName)' != 'Microsoft.Management.UI.Internal'">
```

---

I'd prefer the spaces. :-)

---

I do agree with that. Does `dotnet` recognise the property correctly with them? I'm not sure XML considers it valid formatting, is all. 🙂

---

`dotnet` recognises. Working on the PR I reviewed many *.targets and *.props files in SDK. You can find it in `C:\Program Files\dotnet\sdk\5.0.100-preview.2.20176.6\` folder and subfolders.